docs/index.asciidoc

```diff
@@ -7,15 +7,38 @@ include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 [[ecs-reference]]
 == Elastic Common Schema (ECS) Reference
 
-The Elastic Common Schema (ECS) defines a common set of fields for
-ingesting data into Elasticsearch. A common schema helps you correlate
-data from sources like logs and metrics or IT operations
-analytics and security analytics.
+[float]
+=== What is ECS?
+
+The Elastic Common Schema (ECS) defines a common set of fields,
+their datatype, and gives guidance on their correct usage.
+ECS is used to improve uniformity of event data ingested into Elasticsearch.
```

Comment:
Can we make sure this definition is the same as that used in the glossary?

Reply:
Yes, agreed.

I'm this close to creating a YAML file that's not for the schema, but for text snippets we reuse often...

Let's iterate on the definition in the "glossary" conversation of this PR, and I'm adding a TODO to align it with index.asciidoc